pkg/ansible/runner/eventapi/types.go

```diff
@@ -74,3 +97,31 @@ type StatsEventData struct {
 	Failures     map[string]int `json:"failures"`
 	Skipped      map[string]int `json:"skipped"`
 }
+
+// FailureMessages - failure messages from the event api
+type FailureMessages []string
+
+// GetMessages - retrieves the messages as a string
+func (f FailureMessages) GetMessages() string {
```

Comment:
I might be missing something, but could this whole method be replaced with `strings.Join(f, "\n")` ?